src/Native/System.Security.Cryptography.Native/pal_evp_pkey.h

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include "pal_types.h"
+
+#include <openssl/evp.h>
+
+/*
+Shims the EVP_PKEY_new method.
+
+Returns the new EVP_PKEY instance.
+*/
+extern "C" EVP_PKEY* EvpPkeyCreate();
+
+/*
+Cleans up and deletes a EVP_PKEY instance.
+
+Implemented by calling EVP_PKEY_free.
+
+No-op if pkey is null.
+The given EVP_PKEY pointer is invalid after this call.
+Always succeeds.
+*/
+extern "C" void EvpPkeyDestroy(EVP_PKEY* pkey);
+
+/*
+Function:
+UpRefEvpPkey
+
+Used by System.Security.Cryptography.X509Certificates' OpenSslX509CertificateReader when
```

Comment:
I'd alter these to match the style of the rest of the shims; and get rid of the CoreClrPal notion of who cares about it.